Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -6954,6 +6954,43 @@ bool consumeRadii(std::array<RefPtr<CSSValue>, 4>& horizontalRadii, std::array<R
     return true;
 }
 
+static std::optional<std::array<RefPtr<CSSValuePair>, 4>> consumeBorderRadius(CSSParserTokenRange& args, const CSSParserContext& context)
+{
+    std::array<RefPtr<CSSValuePair>, 4> radii;
+    if (consumeIdent<CSSValueRound>(args)) {
```

Comment:
```suggestion
    if (consumeIdentRaw<CSSValueRound>(args)) {
```
Since you don't need the CSSValue, consumeIdentRaw will be faster.

Also writing this as an early return would be nice:

```
if (!consumeIdentRaw<CSSValueRound>(args))
  return std::nullopt;
```